app/templates/custom/dynamic_pagination/_macro.njk

```diff
@@ -0,0 +1,130 @@
+{% macro onsPagination(params) %}
+
+   {% if params.called_from_index == True %}
+        {% if params.last_page_number|float %}
+            {% set last_page = (params.last_page_number|int + 1) %}
+        {% else %}
+            {% set last_page = params.last_page_number|int %}
+        {% endif %}
+
+        {% set count = [0] %}
+
+        {% set pagination_array = [] %}
+
+        {% for x in range(last_page|int) %}
+            {% if x == 0 %}
+                {% set pagination_array = [{"page": 1, "called_from_index": params.called_from_index, "current": false}] %}
+            {% elif x == params.page_number %}
+                {% set pagination_array = pagination_array.append({"page": params.page_number, "current": true}) %}
```

Comment:
Python's `append` does not seem to return a value. This is almost certainly an error.

Reply:
There are quite a few instances where this is used.

Reply:
Interesting, these lines are copied straight from the previous custom  pagination, so I'm not quite sure how this is  meant to work!